explorer/README.md

```diff
@@ -0,0 +1,54 @@
+# Explorer
+
+Explorer website.
+
+## TOC
+
+- [Explorer](#explorer)
+  - [TOC](#toc)
+  - [Build system and dev environment](#build-system-and-dev-environment)
+  - [Environment variables](#environment-variables)
+    - [NPM scripts](#npm-scripts)
+  - [Running a local Rusk node](#running-a-local-rusk-node)
+
+## Build system and dev environment
+
+The build system assumes that you have at least Node.js v18.x installed. The LTS version is 18.16.0 at the time of writing.
+
+All terminal commands assume that you are positioned in root folder of the repository.
+Run `npm install` from the root folder to get the necessary dependencies.
+
+As the application uses the [Web Crypto API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API), in development mode [`@vitejs/plugin-basic-ssl`](https://github.com/vitejs/vite-plugin-basic-ssl) is used to create a self-signed certificate to run the application in HTTPS. Being the certificate self-signed you need to create an exception in the browser to allow it to use the certificate.
```

Comment:
Maybe you don't want the hassle if you don't use the Crypto API, or need HTTPS for other reasons.